src/libs/README.md

```diff
@@ -0,0 +1,9 @@
+Jsoncons
+--------
+The library chosen to obtain the value of a metric from the json of cube is ` Jsoncons <https://github.com/danielaparker/jsoncons>`_ . It allows to use JsonPath.
```

Comment:
I would also add how to compile this library.
In case this makefile is called automatically from the main makefile, please say that explicitely, so that the user doesn't have to bother about how to compile this code.

Reply:
Done :+1: